index.js

```diff
@@ -89,6 +111,9 @@ exports.handleMessage = function ( hook, context, callback )
   if (context.message.type == 'COLLABROOM' && context.message.data.type == 'RTC_MESSAGE') {
     handleRTCMessage(context.client, context.message.data.payload);
     callback([null]);
+  } else if (context.message.type == 'STATS' && context.message.data.type == 'RTC_MESSAGE') {
```

Comment:
Should this be more clear that it's from ep_webrtc?  I just wonder if someone sees it they will wonder what context the message is from?  Minor issue for sure.

Reply:
I can totally live with it.  Maybe we make an additional issue to deal with this once we're live/tested